tests/log-in.test.js

```diff
@@ -0,0 +1,43 @@
+import LoginPage from '../pageobjects/login.page'
+
+describe('Banking', () => {
```

Comment:
This is supposed to be the name of the suite of login related tests. Consider a more descriptive name.